ietf-optical-impairment-topology.tree

```diff
@@ -56,30 +54,27 @@ module: ietf-optical-impairment-topology
     |        |              |       decimal64
     |        |              +--ro max-chromatic-dispersion?
     |        |              |       decimal64
-    |        |              +--ro chromatic-dispersion-penalty* []
-    |        |              |  +--ro chromatic-dispersion    union
-    |        |              |  +--ro penalty-value           union
-    |        |              +--ro polarization-dispersion-penalty* []
-    |        |              |  +--ro polarization-mode-dispersion
-    |        |              |  |       union
-    |        |              |  +--ro penalty-value
-    |        |              |          union
+    |        |              +--ro cd-penalty* []
+    |        |              |  +--ro cd-value         union
+    |        |              |  +--ro penalty-value    union
+    |        |              +--ro pmd-penalty* []
+    |        |              |  +--ro pmd-value        union
+    |        |              |  +--ro penalty-value    union
     |        |              +--ro max-diff-group-delay?
     |        |              |       uint32
-    |        |              +--ro max-pol-dependent-loss-penalty* []
-    |        |              |  +--ro max-polarization-dependent-loss
-    |        |              |  |       power-in-db-or-null
-    |        |              |  +--ro penalty-value
-    |        |              |          union
+    |        |              +--ro max-pdl-penalty* []
+    |        |              |  +--ro max-pdl-value
```

Comment:
I'd like to remove the max here because it is misleading (and not similar to pmd, cd penalties):
and we miss a max-pdl attribute:
    |        |              +--ro max-polarization-dependant-loss ? loss-in-db-or-null
    |        |              +--ro pdl-penalty* []
    |        |              |  +--ro pdl-value   loss-in-db-or-null
    |        |              |  +--ro penalty-value    union

Reply:
Addressed in https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext-RFC9093-bis/pull/70/commits/0038f5e771db23c9e2109f149b30d3226340db67 and in https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext-RFC9093-bis/pull/70/commits/54a7c3383badc1edf34645c3bb71e0f4a9775b19

Co-authored-by: sergio belotti <sergio.belotti@nokia.com>